packages/wallet/test/Transaction/createTransactionInternals.test.ts

```diff
@@ -1,6 +1,6 @@
 import { Cardano, WalletProvider } from '@cardano-sdk/core';
 import { CreateTxInternalsProps, createTransactionInternals } from '../../src/Transaction';
-import { SelectionConstraints } from '@cardano-sdk/util-dev';
+import { SelectionConstraints } from '../../../cip2/test/util';
```

Comment:
@rhyslbw this is what I'm most unsure about in this PR. I added `cip2/test/util` under tsconfig {includes} for this to work and it's the only occurrence of such pattern in entire sdk.

Reply:
Can you not just add `../../cip2/test` as a project reference?

Reply:
Might be able to do that, but I'm not sure it's any better. `cip2-dev` package as you suggested would be the cleanest solution, but also feels like an overkill for what it would do.

Reply:
> Might be able to do that, but I'm not sure it's any better.

Project references establish the build requirements for the current tsconfig scope.

Reply:
Converted to project reference in 0e58c2f4064381de144033abbcc7fed341f7f3e4